packages/preferences/src/browser/preference-frontend-module.ts

```diff
@@ -17,43 +17,32 @@
 import '../../src/browser/style/index.css';
 
 import { ContainerModule, interfaces } from 'inversify';
-import { bindViewContribution, WidgetFactory, FrontendApplicationContribution } from '@theia/core/lib/browser';
-import { PreferencesContribution } from './preferences-contribution';
-import { createPreferencesTreeWidget } from './preference-tree-container';
-import { PreferencesMenuFactory } from './preferences-menu-factory';
+import { FrontendApplicationContribution, bindViewContribution } from '@theia/core/lib/browser';
```

Comment:
please remove all file moves unrelated to changes of widgets, we have other PRs which will have conflicts